src/lib/consensus/proof_of_stake.ml

```diff
@@ -1412,6 +1412,36 @@ module Make (Inputs : Inputs_intf) : Intf.S = struct
       ~previous_state_hash:Protocol_state.(hash negative_one)
       ~blockchain_state:Snark_transition.(blockchain_state genesis)
       ~consensus_state
+
+  let to_unix_timestamp recieved_time =
+    recieved_time |> Time.to_span_since_epoch |> Time.Span.to_ms
+    |> Unix_timestamp.of_int64
+
+  let%test "Receive a valid consensus_state with a bit of delay" =
+    let {Consensus_state.curr_epoch; curr_slot; _} = Consensus_state.genesis in
```

Comment:
but if the only thing you're testing here is the time then it should be okay

Reply:
I think it is fine to work with genesis as the valid block. It was the easiest consensus state to bootstrap from since there is no `gen` for consensus_state as of now in this branch.